pkg/replication/replication.go

```diff
@@ -255,7 +259,7 @@ func (r *replication) Enact() {
 
 				go func() {
 					defer close(exitCh)
-					err = r.updateOne(node, timeoutCh, aggregateHealth)
```

Comment:
I wonder if there's a linter we can turn on for "this goroutine captures free vars!" It's a dangerous property.